osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/application/services/ActionMapperService.java

```diff
@@ -204,7 +205,6 @@ private void postConstruct() {
         CLASS_TO_MAPPER_MAP.put(ActivityCalendarData.class, this.configurationMapper);
         CLASS_TO_MAPPER_MAP.put(SetConfigurationObjectRequestData.class, this.configurationMapper);
         CLASS_TO_MAPPER_MAP.put(SetAlarmNotificationsRequestData.class, this.configurationMapper);
-        CLASS_TO_MAPPER_MAP.put(SetConfigurationObjectRequestData.class, this.configurationMapper);
```

Comment:
Removed duplicate map key. The rest of the changes in this class are reformatting.